shoppinglist/src/app/view-item/view-item.component.html

```diff
@@ -0,0 +1,7 @@
+<app-add-item (newItemEvent)="addNewItem($event)" [count]="countvalue"></app-add-item>
```

Comment:
move this add item component as a child of app component

Reply:
modified.